arbeitszeit_flask/templates/macros/transactions.html

```diff
@@ -38,7 +38,7 @@
             <p>
                 <strong class="is-size-5">
                     {{ type }}
-                </strong><span class="icon"><i class='{{ peer_type_icon }}'></i></span><span>{{ peer_name }}</span>
+                </strong><span class="icon">'{{ peer_type_icon|icon }}'</span><span>{{ peer_name }}</span>
```

Comment:
This raises an error, when one tries to visit the "account prd" page of a company (e.g. http://127.0.0.1:5000/user/company/73880350-173b-472f-9c89-6278ff4f7293/account_prd):

```
raise FileNotFoundError(f'Icon "{icon_name}" not found: {str(e)}')
FileNotFoundError: Icon "" not found: [Errno 2] No such file or directory: 'arbeitszeit_flask/templates/icons/.html'
```

Reply:
Fixed, `icon_filter` now treats empty strings as intentionally set null values